.github/workflows/reusable-execute-submodule-tests.yaml

```
@@ -45,7 +45,7 @@ jobs:
             terraform_wrapper: false 
             
         - name: Run linter
-          uses: golangci/golangci-lint-action@9fae48acfc02a90574d7c304a1758ef9895495fa # v7.0.1
+          uses: golangci/golangci-lint-action@4afd733a84b1f43292c63897423277bb7f4313a9 # v8.0.0
           if: false
           with:
             version: v1.59
```

requires version 2.1
```suggestion
            version: v2.1
```